docs/index.md

```
@@ -17,6 +17,7 @@ NICo is open source under the Apache 2.0 license.
 ## Quick Links
 
 - [Hardware Compatibility List](hcl.md) — Supported servers and DPUs
-- [Release Notes](/infra-controller/changelog) — What's new in each version
+{/* <!-- rumdl-disable-next-line --> */}
```

markdownlint-cli2 (0.23.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)